connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/webhook/WebhookConnectorRegistry.java

```diff
@@ -16,16 +16,29 @@
  */
 package io.camunda.connector.runtime.inbound.webhook;
 
+import io.camunda.connector.api.inbound.Activity;
+import io.camunda.connector.api.inbound.Severity;
 import io.camunda.connector.runtime.inbound.lifecycle.ActiveInboundConnector;
 import io.camunda.connector.runtime.inbound.webhook.model.CommonWebhookProperties;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Optional;
+import java.util.regex.Pattern;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class WebhookConnectorRegistry {
 
+  private static final String WEBHOOK_CONNECTOR_REGISTRY = "WebhookConnectorRegistry";
+
+  // Reflect changes to this pattern in webhook element templates
+  private static final Pattern CURRENT_WEBHOOK_PATH_PATTERN =
+      Pattern.compile("^[a-zA-Z0-9]([a-zA-Z0-9_-]*[a-zA-Z0-9])?$");
+  private static final String DEPRECATED_WEBHOOK_MESSAGE_PREFIX = "Deprecated webhook path: ";
+  private static final String DEPRECATED_WEBHOOK_MESSAGE_SUFFIX =
+      ". This may lead to unexpected behavior. Consider adjusting the path to match the pattern: "
+          + CURRENT_WEBHOOK_PATH_PATTERN;
+
```

Comment:
We could move this code and the static method in some WebhookValidation utility class to reduce the noise in this class but its rather a cosmetic change and not a blocker.

Reply:
Fair point - I'll circle back to this one once I get clarity on the other remaining comments.

That way, I can avoid auto-resolving (or changing references to) any existing comments on these static fields.

Reply:
@sbuettner I extracted the static constants and method into a new class `WebhookConnectorValidationUtil`.

What do you think?